pyiceberg/manifest.py

```diff
@@ -182,6 +182,7 @@ def __repr__(self) -> str:
             doc="Splittable offsets",
         ),
         NestedField(field_id=140, name="sort_order_id", field_type=IntegerType(), required=False, doc="Sort order ID"),
+        NestedField(field_id=141, name="spec_id", field_type=IntegerType(), required=False, doc="Partition spec ID"),
```

Comment:
I don't think we want to add the fields here since the `spec_id` isn't read/written.